pynumaflow/shared/server.py

```diff
@@ -181,6 +181,7 @@ def start_multiproc_server(
     serv_info = ServerInfo(
         protocol=Protocol.TCP,
         language=Language.PYTHON,
+        minimum_numaflow_version=MINIMUM_NUMAFLOW_VERSION,
         version=get_sdk_version(),
         metadata=get_metadata_env(envs=METADATA_ENVS),
```

Comment:
@kohlisid how come metadata is missing in the other ServerInfo?

Reply:
if all the args are the same for all the invocations, can we just have default values for all the params to `__init__`?

Reply:
Metadata was written only for multiproc connections because we send the multiple ports that we need to connect to. But looking into the code right now there is some scope of refactor where we can reduce the redundancy. I will look into this once and share with @ayildirim21. We can make the changes as part of this PR or a different one as required.